Gems/Atom/RPI/Code/Include/Atom/RPI.Public/FeatureProcessor.h

```diff
@@ -51,6 +54,8 @@ namespace AZ
 
             struct SimulatePacket
             {
+                AZ::Job* m_parentJob = nullptr;
+                // AZ::TaskGraphEvent* m_parentTGE;
```

Comment:
Is this supposed to be commented out or should it be removed? the forward declaration added above should also be removed if this is removed.

Reply:
I removed the comment out code.